CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### ** Not yet released **
 
+* Refreshed the Web and Teleport Connect UI design [#46812](https://github.com/gravitational/teleport/pull/46812)
```

Comment:
```suggestion
* Refreshed the Web UI and Teleport Connect UI design [#46812](https://github.com/gravitational/teleport/pull/46812)
```
Suggested to keep the phrase "Web UI" together across the docs so we have a consistent product naming standard.